src/components/MIDIStateDisplay.tsx

```diff
@@ -42,12 +42,10 @@ const KeyNoteVelocityVisualise = React.memo(
           <p>Value</p>
           <>
             <chakra.label
-              index={1}
-              noChildren={entry.notes?.length ?? 0}
               gridRow={`2 / span ${(entry.notes?.length ?? 0) + 1}`}
               htmlFor={key}
             >
-              {HIDCodes[parseInt(key)]}
+              {entry.notes?.length ?? HIDCodes[parseInt(key)]}
             </chakra.label>
```

Comment:
This change doesn't make much sense to me, it doesn't really seem equivalent?

Reply:
@simon-wh you're right, it doesn't make any sense. Sorry for the long absence - in my defense I became a father a few months ago. I'll rework this PR next weekend.